src/components/SettingsForm.css

```diff
@@ -19,6 +18,15 @@
   transition: 0.3s ease-in-out;
```

Comment:
right now the border is being removed without transition, if you want a transition also when leaving the input, you can put it into the `.form-component` class and define the transition that it only counts for the border-bottom styling.

Reply:
I will look into it! Tks!